pubsub/topics/main.go

```diff
@@ -125,6 +147,34 @@ func publish(client *pubsub.Client, topic, msg string) error {
 	return nil
 }
 
+func publishWithSettings(client *pubsub.Client, topic string, msg []byte) error {
+	ctx := context.Background()
+	// [START publish_settings]
+	t := client.Topic(topic)
+	t.PublishSettings = pubsub.PublishSettings{
+		ByteThreshold:  5000,
+		CountThreshold: 10,
+		DelayThreshold: 100 * time.Millisecond,
+	}
+	result := t.Publish(ctx, &pubsub.Message{Data: msg})
```

Comment:
also, if comments are a thing in these samples, maybe a comment that this kicks of many goroutines?
or do you think that's an impl detail?